app/templates/stories/view.html

```diff
@@ -29,27 +29,27 @@ <h1>{{ story.activist_first }} {{ story.activist_last }}</h1>
                                 <iframe class="youtube-player" type="text/html" src="{{ video_url }}" frameborder="0"
                                             allowFullScreen></iframe>
                             </div>
-                            <p>{{ story.content }}</p>
+                            <p class = "request-description-text" >{{ story.content | safe }}</p>
```

Comment:
Update class names to `story-content-text`

Reply:
done